res/css/categories.css

```diff
@@ -9,7 +9,7 @@
  * They're global css classes that can be used across the project.
  */
 .category-color-transparent {
-  border-color: rgba(0, 0, 0, 0.4);
+  border-color: rgb(0 0 0 / 0.4);
```

Comment:
This is [color-function-notation](https://stylelint.io/user-guide/rules/list/color-function-notation/). I checked that [all our supported browsers support this new syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()#space-separated_values). I believe it makes it easier to extract colors to variables and apply various opacities on it.